src/renderer/components/NodeDocumentationModal.tsx

```diff
@@ -0,0 +1,492 @@
+                                        gap={2}
+                                        left={0}
+                                        maxH="full"
+                                        overflowY="scroll"
+                                        position="absolute"
+                                        top={0}
+                                    >
+                                        <VStack
+                                            alignItems="left"
+                                            display="block"
+                                            divider={<Divider />}
+                                            h="full"
+                                            maxH="full"
+                                            spacing={2}
+                                            textAlign="left"
+                                            w="full"
+                                        >
+                                            <Box userSelect="text">
+                                                <HStack>
+                                                    <IconFactory
+                                                        accentColor={selectedAccentColor}
+                                                        boxSize={6}
+                                                        icon={selectedSchema.icon}
+                                                    />
+                                                    <Heading
+                                                        size="lg"
+                                                        userSelect="text"
+                                                    >
+                                                        {selectedSchema.name}
+                                                    </Heading>
+                                                </HStack>
+                                                <ReactMarkdown
+                                                    components={ChakraUIRenderer(
+                                                        customMarkdownTheme
+                                                    )}
+                                                >
+                                                    {selectedSchema.description}
+                                                </ReactMarkdown>
+                                            </Box>
+                                            <Box position="relative">
+                                                <Heading
+                                                    mb={1}
+                                                    size="sm"
+                                                    userSelect="text"
+                                                >
+                                                    Inputs
+                                                </Heading>
+                                                {selectedSchema.outputs.length > 0 ? (
```

Comment:
So that's why some nodes were shown as having no inputs.
```suggestion
                                                {selectedSchema.inputs.length > 0 ? (
```

Reply:
LOL